_pages/security/dynamic-scanning.md

```diff
@@ -5,29 +5,58 @@ parent: Security
 
 [OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project) is the tool we use for doing automated vulnerability scanning.
 
+<img src="../../assets/images/zap_home.png" alt="zap home screen"/>
+
 ## Usage
 
-You will need a running application to test, which you will want to be as production-like as possible. Before testing, make sure to let your team know, as they may see a spike in requests and errors.
+You will need a running application to test, which you will want to be as production-like as possible. The 'Attack' scripts shouldn't do anything destructive, but will cause a spike in requests and errors. **Inform your team and #devops before running it on live (not locally-hosted) sites.**
 
 1. [Install the ZAP desktop app](https://github.com/zaproxy/zaproxy/wiki/Downloads).
 1. In the Quick Start tab, enter the URL of your application.
     * While you will need to run against production for the official ATO, you can also try out ZAP running against a local server.
-        * If you're using Rails on Mac, start it with
-
-            ```bash
-            rails server --binding 0.0.0.0
-            ```
```

Comment:
Mind if we keep this in? It's an unusual way of starting Rails, so want to make sure we have a note about it.